### ISSUE TITLE
Security Update - Debian Security Advisory DLA-2563 - OpenSSL CVE-2021-23840 and CVE-2021-23841

### ISSUE THREAD

**Issue by CelsoSantos** [bazelbuild/rules_docker#1768]
# 🐞 bug report

### Affected Rule

At least, `go_image` rule ([here](https://github.com/bazelbuild/rules_docker/blob/master/go/go.bzl)), but since it's an issue with the debian base image, all the `<lang>_rules` should be affected.

### Is this a regression?
No

### Description

Recently there has been a security issue with the openssl versions included with debian 9 and 10, identified by [DLA-2563](https://www.debian.org/lts/security/2021/dla-2563).

Security updates have been issued for the affected library and [GoogleContainerTools/distroless](https://github.com/GoogleContainerTools/distroless) has had the fixes [commited](https://github.com/GoogleContainerTools/distroless/blob/1e4a8bb3ad03f71b572cbcb3bbc25f3fd8d0ff14/package_bundle_amd64_debian10.versions) and [published](https://console.cloud.google.com/gcr/images/distroless/GLOBAL/base-debian10@sha256:728bf8b30a5f1a00ea3cd11b943cc3ec6ddbc4b2ffb21b55e87fc6fb22f34944/details?tab=info).

## 🔥 Exception or Error

Security scanning tools will identify the vulnerability and may reject images

<pre><code>
Artifact:                    IMAGE - quay.io/repo-name/authz_server:1.0.3
  RejectReasons (1)
    RejectReason:       9757f3ae-b5c9-4a55-8af0-76cb760c71eb
      Type:            VULNERABILITY
      Name:            DLA-2563-1
      CVSS Score v3:   7.5
      Severity:        high
      Description:     . Impacted Image File(s): var/lib/dpkg/status.d/openssl
</code></pre>

**Output of `bazel version`:**

<pre>
  <code>
root@cffdf2660ad7:/workspace# bazel --version
bazel 3.6.0
  </code>
</pre>

**Rules_docker version:**

<pre>
  <code>
http_archive(
    name = "io_bazel_rules_docker",
    sha256 = "4521794f0fba2e20f3bf15846ab5e01d5332e587e9ce81629c7f96c793bb7036",
    strip_prefix = "rules_docker-0.14.4",
    urls = ["https://github.com/bazelbuild/rules_docker/releases/download/v0.14.4/rules_docker-v0.14.4.tar.gz"],
)
  </code>
</pre>

**Anything else relevant?**
Can the fix be applied also on versions 0.14 and 0.15 please?

**Comment by gravypod** [comment]
Looks like some of the images we depend on stopped pushing a `:debug` tag. The update_deps.sh script wasn't working because of this. I've opened https://github.com/bazelbuild/rules_docker/pull/1770 to fix the issue. After that is merged would you mind rebasing, rerunning that script, and amending your commit.

Thanks!

**Comment by CelsoSantos** [comment]
@gravypod will do :)

**Comment by CelsoSantos** [comment]
I guess after #1770 this has been addressed and #1769 is no longer needed

**Comment by CelsoSantos** [comment]
Also leaving a note in case anyone ends up here looking at the issue and is stuck in versions prior to latest:

If you can't update to newer versions, I have forked, cherry-picked the fixes and released a 0.14.5 version that has the latest images, which you can find here: https://github.com/CelsoSantos/rules_docker/releases/tag/v0.14.5
